src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MemberDefHandler.java

```diff
@@ -51,13 +53,38 @@ public void checkIndentation() {
             checkType();
         }
         final DetailAST firstNode = getMainAst();
-        final DetailAST lastNode = getVarDefStatementSemicolon(firstNode);
+        DetailAST lastNode = getVarDefStatementSemicolon(firstNode);
 
-        if (lastNode != null && !isArrayDeclaration(firstNode)) {
+        if (lastNode != null) {
+            lastNode = getArrayInitNode(firstNode).orElse(lastNode);
```

Comment:
Seems very weird we get `getVarDefStatementSemicolon` first and then if it has a value, throw it away if `getArrayInitNode` provides us with a better result.

Why aren't we looking at `getArrayInitNode` first and then rely on `getVarDefStatementSemicolon` if the first is null?